renv.lock

```diff
@@ -1725,6 +1725,14 @@
         "stats"
       ]
     },
+    "renv": {
+      "Package": "renv",
+      "Version": "1.1.5",
+      "OS_type": null,
+      "NeedsCompilation": "no",
```

Comment:
local leftover?

Reply:
not sure, isn't the idea that we also always push changes to the lockfiles? If not I can revert this if you prefer.